dropwizard-e2e/src/test/java/com/example/badlog/BadLogTest.java

```diff
@@ -48,20 +45,20 @@ public void teardown() {
     @Test
     public void thatLoggingIsntBrokenOnCleanup() throws Exception {
         BadLogApp.runMe(new String[]{"server"});
-        LOGGER.info("I'm after the test");
+        LOGGER.info("I'm tearDown the test");
         Thread.sleep(100);
 
         assertThat(new String(out.toByteArray(), UTF_8))
             .contains("Mayday we're going down")
-            .contains("I'm after the test");
+            .contains("I'm tearDown the test");
```

Comment:
Looks like after/tearDown suffered the same fate

Reply:
Rolled back